inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -373,11 +374,20 @@ def prepare_files(obj, eng):
     if not obj.files:
         return
 
+    def _define_filename(paper, file_name):
+        """Define the filename in according to the article source"""
```

Comment:
Still isn't it nice to describe what the subfunction does, so that you don't have to reparse it with your eyes everytime?